### PR TITLE
Remote Driver support added #61 #62

### PR DESCRIPTION
**Pull request by SergeyPirogov**

### CODE REVIEW

**Comment by SergeyPirogov** [comment]
@apanashchenko @ArtyomAnohin please look at this. Feel free to comment

**Comment by ArtyomAnohin** [comment]
Looks good!

File: kirk-core/src/main/kotlin/com/automation/remarks/kirk/core/WebDriverFactory.kt [hunk 1]
```diff
@@ -38,17 +45,31 @@ class WebDriverFactory {
...
+        val remoteUrl = configuration.remoteUrl()
+        var capabilities: DesiredCapabilities = DesiredCapabilities()
+        when (browser) {
+            CHROME -> capabilities = DesiredCapabilities.chrome()
+            FIREFOX -> capabilities = DesiredCapabilities.firefox()
+            INTERNET_EXPLORER -> capabilities = DesiredCapabilities.internetExplorer()
+        }
+        return RemoteWebDriver(URI.create(remoteUrl).toURL(), capabilities
+                .merge(getCapabilities())
+                .merge(getOptions()))
```
Comment (apanashchenko) [inline kirk-core/src/main/kotlin/com/automation/remarks/kirk/core/WebDriverFactory.kt]: getOptions() only for Chrome browser.What will happen if remote browser Firefox?

Comment (SergeyPirogov) [inline kirk-core/src/main/kotlin/com/automation/remarks/kirk/core/WebDriverFactory.kt]: Yes, this is bug

**Pull request merged**